### PR TITLE
Update Italian translation; fix a couple of minor typos in the English translation.

### PR DESCRIPTION
**Pull request by loreb**
The Italian translation is almost complete - except a couple of things I wasn't sure of and four messages with no translation in any language.

Corrections etc are welcome (@tafazzi ti importuno a Ferragosto :)

### CODE REVIEW

**Comment by Keriew** [comment]
Hey, thanks for the help! 

I can't comment on the translation, but if you think this is ready for merging, I'll merge it.

**Comment by loreb** [comment]
Just fixed a couple of typos... (if anybody asks, I'll blame the temperature)

I did the translation for a family member (I love C3, but haven't played Augustus yet), so there are a few cases where I wasn't sure of the proper context - I double checked French&Spanish, then for any doubt I did my best to note it in comments.

If it's possible to get tafazzi or any Italian speaker to give it a look, that would be ideal, otherwise imho it's ok to merge, with most of it done it should be easier to submit small corrections.

PS
About "Consilium quantitas/emptor/venditor terrae/maritimae": my Latin is getting a bit rusty, but shouldn't the second word be genitive? eg "quantitatis maritimae" etc

**Comment by gianlore** [comment]
A "GREAT" thanks to those who do all this translation work, I look forward to the possibility of using the options that Augustus offers, I also look forward to the translation of the manual in its entirety.

**Comment by loreb** [comment]
@gianlore you're welcome; the manual looks short enough that translating it shouldn't be too much of an issue, but a couple of things that I missed were
1. I haven't played augustus yet, so I occasionally needed to double check other translations / ask family for context
2. a second pair of eyes, especially to double check after everything is done

If you feel like correcting any mistake in my translation, by all means do so!
Same about the manual (or translate it yourself and I can review, it's the same).

**Comment by crudelios** [comment]
Is anyone willing to double check this? If not I'll merge it anyway, since at least we have some new translated strings.

File: src/translation/italian.c [hunk 1]
```diff
@@ -374,6 +374,347 @@ static translation_string all_strings[] = {
     {TR_BUTTON_GO_TO_SITE, "Vai alla posizione" },
     {TR_RETURN_ALL_TO_FORT, "Ritorna tutti" },
     {TR_OPTION_MENU_COST, "Questo costerà"}
```
Comment (crudelios) [inline src/translation/italian.c:376]:
```suggestion
    {TR_OPTION_MENU_COST, "Questo costerà"},
```

Compile error fix

**Comment by crudelios** [comment]
Merged, if there are issues they can be fixed later.

**Pull request merged**